cmd/lint.go

```diff
@@ -577,3 +589,79 @@ func RenderSummary(rs *model.RuleResultSet, silent bool, totalFiles, fileIndex i
 	}
 
 }
+
+func getFilesToLint(globPattern string, filenames []string) []string {
+	dir, err := os.Getwd()
+	if err != nil {
+		pterm.Error.Println(err)
+		os.Exit(1)
```

Comment:
This should not exit, handle the error in the lint command. Return `([]string, error)` from this function.

Reply:
if the user has specifically asked for `--globbed-files` and it throws an error, they should know about it, however if they have not, then they should expect the current/default behavior.

Reply:
I've never written code to handle these scenarios, so let me know if the changes make sense. Thank you for the feedback!